README.md

```diff
@@ -124,49 +124,35 @@ cargo flamegraph --test test_name
-
-FLAGS:
-        --deterministic          Colors are selected such that the color of a function does not change between runs
-        --dev                    Build with the dev profile
-        --flamechart             Produce a flame chart (sort by time, do not merge stacks)
-    -h, --help                   Prints help information
-    -i, --inverted               Plot the flame graph up-side-down
-        --no-default-features    Disable default features
-        --open                   Open the output .svg file with default program
-        --reverse                Generate stack-reversed flame graph
-        --root                   Run with root privileges (using `sudo`)
-        --no-inline              Disable inlining for perf script because of performance issues
-    -V, --version                Prints version information
-    -v, --verbose                Print extra output to help debug problems
-
-OPTIONS:
-        --bench <bench>                    Benchmark to run
-    -b, --bin <bin>                        Binary to run
-    -c, --cmd <custom-cmd>                 Custom command for invoking perf/dtrace
-        --example <example>                Example to run
-    -f, --features <features>              Build features to enable
-    -F, --freq <frequency>                 Sampling frequency
-        --image-width <image-width>        Image width in pixels
-        --manifest-path <manifest-path>    Path to Cargo.toml
-        --min-width <FLOAT>                Omit functions smaller than <FLOAT> pixels [default: 0.01]
-        --notes <STRING>                   Set embedded notes in SVG
-    -o, --output <output>                  Output file, flamegraph.svg if not present
-    -p, --package <package>                package with the binary to run
-        --palette <palette>                Color palette [possible values: hot, mem, io, red, green, blue, aqua, yellow,
-                                           purple, orange, wakeup, java, perl, js, rust]
-        --post-process                     Run a command to process the folded stacks, taking the input from stdin 
-                                           and outputting to stdout.
-        --skip-after <FUNCTION>            Cut off stack frames below <FUNCTION>; may be repeated [linux only]
-        --test <test>                      Test binary to run (currently profiles the test harness and all tests in the
-                                           binary)
-        --unit-test <unit-test>            Crate target to unit test, <unit-test> may be omitted if crate only has one
-                                           target (currently profiles the test harness and all tests in the binary; test
-                                           selection can be passed as trailing arguments after `--` as separator)
-
-ARGS:
-    <trailing-arguments>...    Trailing arguments are passed to the binary being profiled
+Usage: flamegraph [OPTIONS] [-- <TRAILING_ARGUMENTS>...]
```

Comment:
This updates from using `cargo-flamegraph` to `flamegraph`, I think we should keep the former?

Reply:
Nice catch I've regenerated the usage message using `cargo flamegraph --help`